horace_core/Tobyfit/tobyfit_DGdisk_resconv_init.m

```diff
@@ -5,7 +5,7 @@
 %   >> [ok,mess,lookup]=tobyfit_DGdisk_resconv_init(win)
 %
 % For specific pixels:
-%   >> [ok,mess,lookup]=tobyfit_DGdisk_resconv_init(win,indx)
+%   >> [ok,mess,lookup]=tobyfit_DGdisk_resconv_init(win,ipix)
```

Comment:
Any strong reason for the rename? To me, intuitively, due to Fortran, `ipix` is a counter of number pixels akin to `npix` rather than pixel indices. `pix_ind` may be better if it's being renamed.

Reply:
The reason for the rename was to explicitly include 'pix' in the name. The previous name of 'indx' was as an indicator of a general index which covered a scenario of being an Nx3 array of irun-idet-ien, a case which was never used and is in fact unnecessary. It now only is used as array of pixel indices. Perhaps pix_ind is better, but given that I've made the change and to change again would require numerous edits in several iles I'd rather not now, and chalk this up to using apossibly better name for similar entities in the future.